--- origsrc/gnumeric-1.10.15/doc/Makefile.am	2010-06-04 22:49:20.000000000 -0500
+++ src/gnumeric-1.10.15/doc/Makefile.am	2011-05-24 18:01:43.254161000 -0500
@@ -91,7 +91,7 @@ endif
 
 winhelp_DATA =
 winhelpdir=$(gnumeric_datadir)/$(subdir)
-if WITH_WIN32
+if WITH_NATIVE_WIN32
 winhelp_DATA += gnumeric.chm gnumeric.hhmap
 
 gnumeric.hhmap: chm-stamp
--- origsrc/gnumeric-1.10.15/icons/Makefile.am	2010-06-14 19:50:13.000000000 -0500
+++ src/gnumeric-1.10.15/icons/Makefile.am	2011-05-24 18:01:43.263161500 -0500
@@ -24,7 +24,7 @@ EXTRA_DIST = $(icons_DATA) $(mimeicon_DA
 gtk_update_icon_cache = gtk-update-icon-cache -f -t $(datadir)/icons/hicolor
 
 fake_index_theme_target =
-if WITH_WIN32
+if WITH_NATIVE_WIN32
 fake_index_theme_target += install-fake-index-theme
 
 install-fake-index-theme: fake-index-theme
--- origsrc/gnumeric-1.10.15/src/Makefile.am	2011-05-06 20:16:32.000000000 -0500
+++ src/gnumeric-1.10.15/src/Makefile.am	2011-05-24 18:01:32.884567900 -0500
@@ -364,12 +364,12 @@ if WITH_NATIVE_WIN32
     ssconvert_LDFLAGS	+= -mconsole -Wl,--stack,0x4000000
     ssindex_LDFLAGS	+= -mconsole
     ssgrep_LDFLAGS	+= -mconsole
+endif
 
+if WITH_WIN32
     libspreadsheet_la_DEPENDENCIES = libspreadsheet.def
     libspreadsheet_la_LDFLAGS += -no-undefined -export-symbols libspreadsheet.def
-endif
 
-if WITH_WIN32
 libspreadsheet.def: local.def
 	echo EXPORTS > $@ && \
 	cat $^ | sort >> $@
--- origsrc/gnumeric-1.10.15/tools/win32/Makefile.am	2010-07-30 19:41:09.000000000 -0500
+++ src/gnumeric-1.10.15/tools/win32/Makefile.am	2011-05-24 18:01:43.267161700 -0500
@@ -14,7 +14,7 @@ EXTRA_DIST = gnumeric.nsi.in gtkrc modul
 	build jhbuildrc.py
 
 
-if WITH_WIN32
+if WITH_NATIVE_WIN32
 install-data-hook:
 	install gnumeric.nsi ${prefix}
 
