--- origsrc/gnumeric-1.8.3/libspreadsheet.pc.in	2008-04-24 17:43:23.000000000 -0500
+++ src/gnumeric-1.8.3/libspreadsheet.pc.in	2008-05-26 09:23:20.687500000 -0500
@@ -7,12 +7,12 @@
 libgoffice=@LIBGOFFICE@
 
 Name: libspreadsheet
-Description: The spreadsheet internals of Gnumeric (DEVELOPMENT VERSION)
+Description: The spreadsheet internals of Gnumeric
 Version: @VERSION@
 URL: http://www.gnumeric.org
 Requires: glib-2.0 gobject-2.0 libgsf-1 libxml-2.0 gtk+-2.0 libglade-2.0 @LIBGOFFICE@
 Requires.private: libart-2.0
-Libs: -L${libdir} -lspreadsheet-@GNUMERIC_API_VER@ @GTK_EXTRA_LIBS@
+Libs: -L${libdir} -lspreadsheet
 
 # for now include both the directory and the child, until we rename the includes in svn
 Cflags: -I${includedir}/libspreadsheet-@GNUMERIC_API_VER@/spreadsheet -I${includedir}/libspreadsheet-@GNUMERIC_API_VER@ -I${gnumeric_libdir}/include
