--- origsrc/gnumeric-1.8.3/configure.in	2008-04-11 06:00:07.000000000 -0500
+++ src/gnumeric-1.8.3/configure.in	2008-05-26 09:23:20.671875000 -0500
@@ -272,11 +272,19 @@
 mingw* | pw32* | cygwin*)
     with_win32=yes
     GNUMERIC_PLUGIN_LDFLAGS="-no-undefined $GNUMERIC_LIBS -Wl,--enable-runtime-pseudo-relo,--export-all-symbols \$(top_builddir)/src/libspreadsheet.la -s"
+    ;;
+esac
+
+with_native_win32=no
+case $host_os in
+mingw* | pw32*)
+    with_native_win32=yes
     AC_ARG_VAR(WINDRES, [The windres executable (used by win32 builds only).])
     AC_CHECK_TOOL(WINDRES, windres, :)
     ;;
 esac
 AM_CONDITIONAL(WITH_WIN32, test $with_win32 = yes)
+AM_CONDITIONAL(WITH_NATIVE_WIN32, test $with_native_win32 = yes)
 AM_CONDITIONAL(CROSS_COMPILING, test x"$cross_compiling" != "xno")
 
 GNUMERIC_PLUGIN_LDFLAGS="-avoid-version $GNUMERIC_PLUGIN_LDFLAGS"
@@ -908,7 +916,7 @@
      major_ver=`$PYTHON -c 'import sys ; sys.stdout.write(sys.version[0:1])'`]
     if test "x$major_ver" = "x2"; then
 	AC_MSG_RESULT(yes)
-	if test "x$with_win32" = "xyes" ; then
+	if test "x$with_native_win32" = "xyes" ; then
 	    # 2.x on linux, 2x on win32
 	    PY_VERSION=`echo $PY_VERSION | sed -e 's/\\.//'`
 	fi
@@ -923,7 +931,7 @@
 	PY_PREFIX=`$PYTHON -c 'import sys ; sys.stdout.write(sys.prefix)'`
     fi
     if test "x$PY_INCLUDE_DIR" = x; then
-	if test "x$with_win32" = "xyes" ; then
+	if test "x$with_native_win32" = "xyes" ; then
 	    PY_INCLUDE_DIR="$PY_PREFIX/include"
 	else
 	    PY_INCLUDE_DIR=`$PYTHON -c 'import sys ; import distutils.sysconfig ; sys.stdout.write(distutils.sysconfig.get_python_inc())'`
@@ -948,7 +956,7 @@
 AM_CONDITIONAL(WITH_PYTHON, test "x$python_msg" = xyes)
 if test "x$python_msg" = xyes; then
     if test "x$PY_LIB_DIR" = x; then
-	if test "x$with_win32" = "xyes" ; then
+	if test "x$with_native_win32" = "xyes" ; then
 	    PY_LIB_DIR="$PY_PREFIX/libs"
 	else
 	    PY_LIB_DIR=`$PYTHON -c 'import sys ; import distutils.sysconfig ; sys.stdout.write(distutils.sysconfig.get_config_var("LIBPL"))'`
--- origsrc/gnumeric-1.8.3/schemas/Makefile.am	2008-01-14 17:54:25.000000000 -0600
+++ src/gnumeric-1.8.3/schemas/Makefile.am	2008-05-26 09:23:20.703125000 -0500
@@ -21,7 +21,7 @@
 install-data-local:
 endif
 
-if WITH_WIN32
+if WITH_NATIVE_WIN32
 SUFFIXES = .schemas.in .reg .hkcu.reg
 regdir = $(prefix)/etc/win32/reg
 reg_DATA = $(schema_in_files:.schemas.in=.reg) $(schema_in_files:.schemas.in=.hkcu.reg)
--- origsrc/gnumeric-1.8.3/src/Makefile.am	2008-01-14 17:54:39.000000000 -0600
+++ src/gnumeric-1.8.3/src/Makefile.am	2008-05-26 09:23:20.703125000 -0500
@@ -292,10 +292,12 @@
 
 gnumeric_LDFLAGS = -export-dynamic
 
-if WITH_WIN32
+if WITH_NATIVE_WIN32
     gnumeric_LDADD += gnumeric_rc.o
     gnumeric_LDFLAGS += -mwindows
+endif
 
+if WITH_WIN32
     libspreadsheet_la_DEPENDENCIES = libspreadsheet.def
     libspreadsheet_la_LDFLAGS += -no-undefined -export-symbols libspreadsheet.def
 
@@ -390,7 +392,7 @@
 	echo '#endif /* GNUMERIC_PATHS_H */'; \
 	) >$@
 
-if WITH_WIN32
+if WITH_NATIVE_WIN32
 gnumeric_rc.o: gnumeric.rc
 	$(WINDRES) -DVERSION=$(VERSION) -DVERSION_NUMBER=`echo $(VERSION).0.0.0 | sed 's/\([0-9]*\)[^.]*\.*\([0-9]*\)[^.]*\.*\([0-9]*\)[^.]*\.*\([0-9]*\).*/\1,\2,\3,\4/'` --include-dir $(srcdir) -i $< -o $@
 install-exec-hook:
--- origsrc/gnumeric-1.8.3/tools/Makefile.am	2008-03-12 21:30:16.000000000 -0500
+++ src/gnumeric-1.8.3/tools/Makefile.am	2008-05-26 09:23:20.718750000 -0500
@@ -5,7 +5,7 @@
 AM_CPPFLAGS = $(GNUMERIC_CFLAGS) -DGNUMERIC_INTERNAL
 
 noinst_PROGRAMS =
-if WITH_WIN32
+if WITH_NATIVE_WIN32
     noinst_PROGRAMS += gconf-schemas-to-win32-reg
     gconf_schemas_to_win32_reg_SOURCES = gconf-schemas-to-win32-reg.c
     gconf_schemas_to_win32_reg_LDADD = $(GNUMERIC_LIBS)
