--- origsrc/gnumeric-1.9.14/configure.in	2009-09-25 17:46:12.000000000 -0500
+++ src/gnumeric-1.9.14/configure.in	2009-10-30 16:23:40.692944600 -0500
@@ -182,6 +182,18 @@ mingw* | pw32* | cygwin*)
     ;;
 esac
 
+with_native_win32=no
+case $host_os in
+mingw* | pw32*)
+    with_native_win32=yes
+    AC_ARG_VAR(WINDRES, [The windres executable (used by win32 builds only).])
+    AC_CHECK_TOOL(WINDRES, windres, :)
+    ;;
+esac
+AM_CONDITIONAL(WITH_WIN32, test $with_win32 = yes)
+AM_CONDITIONAL(WITH_NATIVE_WIN32, test $with_native_win32 = yes)
+
+
 dnl *******************
 dnl Should we use gtk ?
 dnl *******************
@@ -205,7 +217,7 @@ if test "x$gnumeric_with_gtk" = xtrue ; 
 fi
 
 gnumeric_with_gnome=false
-if test "x$gnumeric_with_gtk" = xtrue -a "x$with_win32" = xno ; then
+if test "x$gnumeric_with_gtk" = xtrue -a "x$with_native_win32" = xno ; then
 	dnl ************************************
 	dnl Are the GNOME extensions available ?
 	dnl ************************************
@@ -249,20 +261,10 @@ PKG_CHECK_MODULES(GNUMERIC, $gnumeric_re
 AC_SUBST(GNUMERIC_LIBS)
 AC_SUBST(GNUMERIC_CFLAGS)
 
-with_native_win32=no
-case $host_os in
-mingw* | pw32*)
-    with_native_win32=yes
-    AC_ARG_VAR(WINDRES, [The windres executable (used by win32 builds only).])
-    AC_CHECK_TOOL(WINDRES, windres, :)
-    ;;
-esac
-AM_CONDITIONAL(WITH_WIN32, test $with_win32 = yes)
-AM_CONDITIONAL(WITH_NATIVE_WIN32, test $with_native_win32 = yes)
 AM_CONDITIONAL(CROSS_COMPILING, test x"$cross_compiling" != xno)
 
 if test "x$with_win32" = "xyes"; then
-    GNUMERIC_PLUGIN_LDFLAGS="-avoid-version -no-undefined $GNUMERIC_LIBS -Wl,--enable-runtime-pseudo-relo,--export-all-symbols \$(top_builddir)/src/libspreadsheet.la -s"
+    GNUMERIC_PLUGIN_LDFLAGS="-avoid-version -no-undefined $GNUMERIC_LIBS -Wl,--enable-runtime-pseudo-reloc,--export-all-symbols \$(top_builddir)/src/libspreadsheet.la -s"
 else
     GNUMERIC_PLUGIN_LDFLAGS="-avoid-version"
 fi
--- origsrc/gnumeric-1.9.14/src/Makefile.am	2009-09-24 18:49:51.000000000 -0500
+++ src/gnumeric-1.9.14/src/Makefile.am	2009-10-30 16:45:48.881830800 -0500
@@ -363,12 +363,12 @@ if WITH_NATIVE_WIN32
     ssconvert_LDFLAGS	+= -mconsole
     ssindex_LDFLAGS	+= -mconsole
     ssgrep_LDFLAGS	+= -mconsole
+endif
 
+if WITH_WIN32
     libspreadsheet_la_DEPENDENCIES = libspreadsheet.def
     libspreadsheet_la_LDFLAGS += -no-undefined -export-symbols libspreadsheet.def
-endif
 
-if WITH_WIN32
 libspreadsheet.def: local.def 
 	echo EXPORTS > $@ && \
 	cat $^ | sort >> $@
